packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -100,7 +100,7 @@ If false the selection should not be updated when child blocks specified in the
 ### `templateLock`
 * **Type:** `String|Boolean`
 
-Template locking of `InnerBlocks` is similar to [Custom Post Type templates locking](https://wordpress.org/gutenberg/handbook/templates/#locking).
+Template locking of `InnerBlocks` is similar to [Custom Post Type templates locking](https://developer.wordpress.org/block-editor/developers/block-api/block-templates/#locking).
```

Comment:
Shouldn’t it use GitHub based link instead? It would be converted accordingly in the handbook and it would still work on GH.

The same applies to another updated link in the same file.